src/main/resources/config/application.yml

```diff
@@ -147,7 +147,7 @@ eureka:
     instance-info-replication-interval-seconds: 10
     registry-fetch-interval-seconds: 10
     service-url:
-      defaultZone: http://admin:${spring.security.user.password:admin}@localhost:${server.port}/eureka/
+      defaultZone: https://admin:${spring.security.user.password:admin}@jhipsterproject.appspot.com/eureka/
```

Comment:
Are you sure? This will be applied for all apps.

Reply:
@PierreBesson : Thanks for pointing it out. It was a mistake. I was testing the application and missed migrating this to the `application-prod-gae.yml`. Done. :smile: